maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -30852,8 +30852,7 @@
 "cbo" = (
 /obj/structure/machinery/door/airlock/almayer/maint{
 	req_access = null;
-	req_one_access = null;
-	req_one_access_txt = "21"
+	req_one_access = list(2,21)
```

Comment:
please use `req_one_access_txt` for this, list instances in map files are deprecated

Reply:
Alright, fixed!